src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/ace-uncompressed.js

```diff
@@ -9696,81 +9696,83 @@ oop.inherits(Mode, CMode);
 exports.Mode = Mode;
```

Comment:
It's generated when we sync to ace. Note it's also not used (this is just
here in case we want to run unminified for debugging).

Yes, the rules you added will indeed take precedence. There is one more
place you need to update though:

https://github.com/rstudio/rstudio/blob/master/src/gwt/src/org/rstudio/studio/client/common/reditor/EditorLanguage.java#L67-L68

We need to change this reference to "mode/sql" rather than "ace/mode/sql"
(and move it up with the other modes we have our own definitions for).

@kevinushey are there any other changes required here?


On Wed, Jul 13, 2016 at 12:16 PM, Javier Luraschi <notifications@github.com>
wrote:

> In
> src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/ace-uncompressed.js
> <https://github.com/rstudio/rstudio/pull/730#discussion_r70657747>:
>
> > @@ -9696,81 +9696,83 @@ oop.inherits(Mode, CMode);
> >  exports.Mode = Mode;
>
> @jjallaire <https://github.com/jjallaire> @kevinushey
> <https://github.com/kevinushey> is it generated by rstudio? How should I
> regenerate this? If not, should the rules I've added take precedence over
> this file? Currently there are not, so I'll take a closer look.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/rstudio/rstudio/pull/730/files/532895f84dc8032da36fd784224114266c0ce4b7#r70657747>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGXx7wWOhLZQK4F5GRebPrlqZGnSlDPks5qVQ9tgaJpZM4JLGni>
> .
>